library/std/src/panicking.rs

```diff
@@ -699,7 +699,11 @@ fn rust_panic_with_hook(
         // have limited options. Currently our preference is to
         // just abort. In the future we may consider resuming
         // unwinding or otherwise exiting the thread cleanly.
-        rtprintpanic!("thread panicked while panicking. aborting.\n");
+        if !can_unwind {
+            rtprintpanic!("this is a non-unwinding panic. aborting.\n");
```

Comment:
This is written from a different perspective than the unwinding case. Is that because this reads differently in context?

Reply:
Is it? What alternative wording would preserve the same perspective?

Reply:
"non-unwinding panic launched. aborting."

Reply:
Ah I see. Seems fine for me, except I don't think "launch" is usually the verb we use for panics, is it?

The docs usually seem to use "panic" as verb and not noun, so... not sure how to best use that here.

Reply:
I am going with "thread caused non-unwinding panic." for now. Thoughts?

Reply:
Yup, that reads better to me.